### PR TITLE
Add new Yellowed theme to default themes

### PR DESCRIPTION
**Pull request by Gael-Lopes-Da-Silva**
This is Yellowed, a theme for vscode (you can find it [here](https://github.com/Gael-Lopes-Da-Silva/YellowedVSCode)) that I ported to Helix ([github](https://github.com/Gael-Lopes-Da-Silva/YellowedHelix)).

### CODE REVIEW

File: runtime/themes/yellowed.toml [hunk 1]
```diff
@@ -0,0 +1,135 @@
...
+"ui.cursor.match"                 = { bg = "light_gray" }
+"ui.cursor.primary"               = { bg = "yellow", fg = "black" }
+"ui.cursor.primary.normal"        = { bg = "yellow", fg = "black" }
+"ui.cursor.primary.insert"        = { bg = "yellow", fg = "black" }
+"ui.cursor.primary.select"        = { bg = "yellow", fg = "black" }
+"ui.debug.breakpoint"             = { fg = "red" }
+"ui.debug.active"                 = { fg = "green" }
+"ui.gutter"                       = { fg = "light_gray" }
+"ui.gutter.selected"              = { fg = "light_gray" }
+"ui.highlight.frameline"          = { fg = "cursor_line" }
```
Comment (the-mikedavis) [inline runtime/themes/yellowed.toml:70]: It looks like this color doesn't exist in the palette

File: runtime/themes/yellowed.toml [hunk 2]
```diff
@@ -0,0 +1,135 @@
...
+yellow      = "#ffd900"
+gray        = "#2a2a2a"
+dark_gray   = "#242424"
+light_gray  = "#545454"
+purple      = "#994c92"
+blue        = "#008DFF"
+menu        = "#202020"
+selection   = "#3f3f3f"
+
+# syntaxe
```
Comment (the-mikedavis) [inline runtime/themes/yellowed.toml:126]:
```suggestion
# syntax
```

File: runtime/themes/yellowed.toml (hunk 1 above)
Comment (Gael-Lopes-Da-Silva) [reply runtime/themes/yellowed.toml]: Old color, fixed

File: runtime/themes/yellowed.toml (hunk 2 above)
Comment (Gael-Lopes-Da-Silva) [reply runtime/themes/yellowed.toml]: Typo fixed

**Review by the-mikedavis — APPROVED** [review]

**Review by pascalkuthe — APPROVED** [review]

**Pull request merged**